exonum-node/src/proto/consensus.proto

```diff
@@ -18,31 +18,16 @@ syntax = "proto3";
 
 package exonum.consensus;
 
+// TODO: is it needed?
```

Comment:
I think since we put it in every file, there is no harm to have this as part of coding standard.